fe/fe-core/src/main/java/com/starrocks/privilege/DbPEntryObject.java

```diff
@@ -23,13 +23,17 @@
 import java.util.Objects;
 
 public class DbPEntryObject implements PEntryObject {
-    public static final long ALL_DATABASE_ID = -2; // -2 represent all
+    public static final String ALL_DATABASES_UUID = "ALL_DATABASES_UUID"; // -2 represent all
```

Comment:
why this change to string? what comment "-2 represent all" means?

Reply:
In order to be in compatible with external table, see the comments for this pr. the comments need to be modified, i will change it.